controller/src/beerocks/master/db/db.cpp

```diff
@@ -2495,7 +2495,7 @@ bool db::is_client_in_persistent_db(const sMacAddr &mac)
 }
 
 bool db::add_client_to_persistent_db(const sMacAddr &mac,
-                                     std::unordered_map<std::string, std::string> params)
+                                     const std::unordered_map<std::string, std::string> &params)
```

Comment:
Please don't add a separate commit to fix Cppcheck issues.
Fix it directly on the commits you added the code which raised the issues.

Reply:
will fix